lua/lv-which-key/init.lua

```diff
@@ -187,10 +148,24 @@ local mappings = {
   },
   T = {
     name = "Treesitter",
-    i = { ":TSConfigInfo<cr>", "Info" },
+    i = { "<cmd>TSConfigInfo<cr>", "Info" },
   },
 }
 
+local vopts = {
+  mode = "v", -- VISUAL mode
+  prefix = "<leader>",
+  buffer = nil, -- Global mappings. Specify a buffer number for buffer local mappings
+  silent = true, -- use `silent` when creating keymaps
+  noremap = true, -- use `noremap` when creating keymaps
+  nowait = false, -- use `nowait` when creating keymaps
+}
+-- NOTE: Prefer using : over in the command expression as the later avoids going back in normal-mode.
+-- see https://neovim.io/doc/user/map.html#:map-cmd
+local vmappings ={
+  ["/"] = { "<cmd>CommentToggle<CR>", "Comment" },
```

Comment:
As according to the note, you should use : instead of \<cmd> :smiley: 
It doesn't work otherwise